AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -542,6 +533,26 @@ - (void)resumeWithIdentifyingObjectSync:(id <NSObject>)identifyingObject {
                             }];
 }
 
+- (void)pauseSendingLogsWithToken:(NSString *)token {
+  NSString *targetKey = [MSUtility targetKeyFromTargetToken:token];
+  @synchronized (self.pausedTargetKeys) {
+    MSLogDebug([MSAppCenter logTag], @"Pause channel for target key %@.", targetKey);
+    [self.pausedTargetKeys addObject:targetKey];
+  }
+}
+
+- (void)resumeSendingLogsWithToken:(NSString *)token {
+  NSString *targetKey = [MSUtility targetKeyFromTargetToken:token];
+  @synchronized (self.pausedTargetKeys) {
+    MSLogDebug([MSAppCenter logTag], @"Resume channel for target key %@.", targetKey);
+    [self.pausedTargetKeys removeObject:targetKey];
+
+    // Update item count and check logs if it meets the conditions to send logs.
+    self.itemsCount = [self.storage countLogs];
```

Comment:
It should be itemCount not itemsCount

Reply:
Yes, totally agree with you but not in the scope so we will refactor separately.